browsers/firefox_android.json

```diff
@@ -37,6 +72,21 @@
           "release_notes": "https://developer.mozilla.org/Firefox/Releases/10",
           "status": "retired"
         },
+        "11": {
+          "release_date": "2012-03-13",
+          "release_notes": "https://developer.mozilla.org/Firefox/Releases/11",
+          "status": "retired"
+        },
+        "12": {
+          "release_date": "2012-04-24",
+          "release_notes": "https://developer.mozilla.org/Firefox/Releases/12",
+          "status": "retired"
+        },
+        "13": {
+          "release_date": "2012-06-05",
+          "release_notes": "https://developer.mozilla.org/Firefox/Releases/13",
```

Comment:
Should maybe be [`https://www.mozilla.org/firefox/android/13.0beta/releasenotes/`](https://www.mozilla.org/firefox/android/13.0beta/releasenotes/).